allauth/socialaccount/providers/authentiq/provider.py

```diff
@@ -0,0 +1,94 @@
+from allauth.account.models import EmailAddress
+from allauth.socialaccount.app_settings import \
+    QUERY_EMAIL, EMAIL_REQUIRED, EMAIL_VERIFICATION
+from allauth.socialaccount.providers.base import AuthAction, ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+
+class Scope(object):
+    NAME = 'aq:name'
+    EMAIL = 'email'
+    PHONE = 'phone'
+    EMAIL_VERIFIED = 'email~rs'
+    PHONE_VERIFIED = 'phone~rs'
+    ADDRESS = 'address'
+    PUSH = 'aq:push'
+
+
+IDENTITY_CLAIMS = frozenset([
+    'sub', 'name', 'given_name', 'family_name', 'middle_name',
+    'nickname', 'preferred_username', 'profile', 'picture',
+    'website', 'email', 'email_verified', 'gender', 'birthdate',
+    'zoneinfo', 'locale', 'phone_number', 'phone_number_verified',
+    'address', 'updated_at'
+])
+
+
+class AuthentiqAccount(ProviderAccount):
+    def get_profile_url(self):
+        return self.account.extra_data.get('profile')
```

Comment:
This is the OIDC `profile` claim, so a URL to the end user's profile. But it isn't returned atm by Authentiq ID.